src/intents/cooktime_intent.js

```diff
@@ -1,10 +1,9 @@
 import {get_recipe, get_user_recipe} from '../mongo_helper'
-import {set_session_data} from '../session_helper'
```

Comment:
Everything in this file should be reverted back except for uncommenting out the declaration of the context var